src/constants/endpoints.js

```diff
@@ -36,4 +36,9 @@ export const endpoints = {
     create: () => `${apiUrl}/folders`,
     delete: (folderId) => `${apiUrl}/folders/${folderId}`,
   },
+  upload: {
+    uploadCreate: () => `${apiUrl}/uploads`,
+    scheduleAnalysis: () => `${apiUrl}/jobs`,
```

Comment:
Shouldn't it be part of `jobs` object?

Reply:
Yes, we have to fix it once we hop on jobs section then we will keep all the related apis to it in that.